runtime/doc/news.txt

```diff
@@ -80,6 +80,9 @@ The following changes to existing APIs or features add new behavior.
 • The `workspace/didChangeWatchedFiles` LSP client capability is now enabled
   by default.
 
+• `vim.loop` has been renamed to `vim.uv`. `vim.loop` will be deprecated and
```

Comment:
I think this fits better under "DEPRECATIONS" -- no need to mention that it will be deprecated than (as it arguably already is).